Task-5/dbconnection/connection.go

```diff
@@ -0,0 +1,49 @@
+package dbconnection
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/joho/godotenv"
+	_ "github.com/lib/pq"
+)
+
+type application struct {
+	database *sql.DB
+}
+
+var app application
+
+// ConnectDatabase function connection the Database to particular database in postgreSQL
+func ConnectDatabase() error {
+
+	var err error
+
+	// Getting map from .env file and using godotenv package
+	envMap, err := godotenv.Read(".env")
```

Comment:
godotenv.Read()
why need .env two times.

change .env reading 1 time and take necessary env variables in 1 read.